scraper_hospital_data.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+"""Script for pulling down CA COVID-19 hospitalization stats"""
+
+import click
+import json
+import logging
+import os
+import traceback
+from pathlib import Path
+from typing import Tuple
+
+from covid19_sfbayarea.data import hospitals
+from covid19_sfbayarea.utils import cli_friendly_county
+
+
+# prep lists of counties the user can choose to filter the data
+# format county strings so they're command line-friendly
+with open("covid19_sfbayarea/counties.json") as f:
+    ca_county_list = json.load(f)
+
+bay_area_counties = [
+    cli_friendly_county(county) for county in ca_county_list.get("Bay Area")
+]
+
+other_ca_counties = [
+    cli_friendly_county(county) for county in ca_county_list.get("Other CA")
+]
+
+all_ca_counties = bay_area_counties + other_ca_counties
```

Comment:
Thanks @Mr0grog. Would you suggest just putting all the county names as a list inside `scraper_hospital_data.py`?

Reply:
Nah, it's big enough that I think it's totally reasonable to put in its own file. It'd just simplify things if it were a `.py` file instead of `.json`.

Reply:
Cool. I've put the two lists in variables inside `covid19_sfbayarea/ca_counties.py`.